src/components/molecules/TodoItem.vue

```diff
@@ -13,30 +13,48 @@ const toggleTodo = (todo) => store.dispatch('toggleTodo', todo)
 </script>
 
 <template>
-  <div class="todo-item">
-    <AppCheckboxInput :checked="props.todo.completed" @change="toggleTodo(todo)" />
-    <div>
-      <span
-        :class="{
+  <a href="#" class="list-group-item d-flex align-items-center">
+    <AppCheckboxInput class="ms-2" :checked="props.todo.completed" @change="toggleTodo(todo)" />
+    <span
+      href="#"
+      :class="[
+        {
           'todo-item-completed': props.todo.completed,
-        }"
-        >{{ props.todo.title }}</span
-      >
-      <AppButton :class="['button', 'item__button--delete']" label="X" @click="removeTodo(props.todo)" />
-    </div>
-  </div>
+        },
+        'ms-4',
+      ]"
+      @click="toggleTodo(todo)"
+      >{{ props.todo.title }}</span
+    >
+    <AppButton
```

Comment:
The label property is required for the AppButton component, but you haven't defined it here